tests/nnfw_api/src/GenModelTests/one_op_tests/DepthwiseConv2D.test.cc

```diff
@@ -19,27 +19,51 @@
 TEST_F(GenModelTest, OneOp_DepthwiseConv2D)
 {
   CircleGen cgen;
-  std::vector<float> weight_data{1, 2, 3, 4, -9, 10, -11, 12, 5, 6, 7, 8, 13, -14, 15, -16};
+  std::vector<float> weight_data{1.0f, 2.0f, 3.0f, 4.0f, -9.0f, 10.0f,  -11.0f, 12.0f,
+                                 5.0f, 6.0f, 7.0f, 8.0f, 13.0f, -14.0f, 15.0f,  -16};
```

Comment:
Why weight shape is {1, 2, 2, 2} and data size is 16? 

```suggestion
  std::vector<float> weight_data{1.0f, 2.0f, 3.0f, 4.0f, -9.0f, 10.0f,  -11.0f, 12.0f};
```